src/main/resources/db/migration/V15_3__add_column_huskelapp_active.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE person_oversikt_status
+ADD COLUMN huskelapp_active BOOLEAN DEFAULT FALSE;
```

Comment:
Tanker om navnet her? Gikk bort ifra `somethingSomethingUbehandlet`-mønsteret fordi vi datamodellen til huskelapp kaller dette `isActive`. Men jeg er åpen for forslag 😊

Reply:
Synes active passer fint ja!

Reply:
Evt. må "active" bety det _samme som_ ubehandlet, sånn at man kunne ha endret navn på alle de eksisterende fra ubehandlet til active også, uten at vi har endret betydning.

Reply:
Hm, du har et poeng der. Meningen (i første omgang i hvert fall) er vel egentlig bare å vise i oversikten om det finnes en aktiv huskelapp på en person eller ei, så kanskje vi kunne droppet ispersonoppgave og heller konsumert huskelapp-topicen direkte her.